### PR TITLE
Fix app config

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: hamlpy/apps.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from django.apps import AppConfig
```
Comment (norkans7) [inline hamlpy/apps.py:1]: This was the problem causing strings to not be well picked by makemessages as this app was not loading well to the `templatize` patch was not being called

File: hamlpy/test/test_compiler.py [hunk 2]
```diff
@@ -221,8 +221,7 @@ def test_python_filter(self):
     def test_pygments_filter(self):
         self._test(":highlight\n", "")  # empty
         self._test(
-            ":highlight\n  print(1)\n",
-            '<div class="highlight"><pre><span></span><span class="k">print</span><span class="p">(</span><span class="mi">1</span><span class="p">)</span>\n</pre></div>',
+            ":highlight\n  print(1)\n", '<div class="highlight"><pre><span></span>print(1)\n</pre></div>'
```
Comment (norkans7) [inline hamlpy/test/test_compiler.py:224]: just because we update `pygments`

File: hamlpy/test/test_templates.py [hunk 3]
```diff
@@ -60,7 +60,7 @@ def test_templates(self):
 
             if not test.passed():
                 print("\nHTML (actual): ")
-                print("\n".join(["%d. %s" % (i + 1, l) for i, l in enumerate(test.actual_html.split("\n"))]))
+                print("\n".join(["%d. %s" % (i + 1, line) for i, line in enumerate(test.actual_html.split("\n"))]))
```
Comment (norkans7) [inline hamlpy/test/test_templates.py:63]: just code style not wanting single letter variable names

**Review by rowanseymour — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -5,8 +5,8 @@ jobs:
     name: Test
     strategy:
       matrix:
-        python-version: ['3.6.x', '3.7.x', '3.8.x']
-        django-version: ['2.1.15', '2.2.17', '3.0.11', '3.1.4']
+        python-version: ['3.8.x', '3.9.x', '3.10.x']
```
Comment (rowanseymour) [inline .github/workflows/ci.yml:8]: better to test 3.9 to 3.11.. we'll be on 3.11 in a year

Comment (norkans7) [reply .github/workflows/ci.yml]: Added 3.11 but just kept 3.8 as well for now

**Pull request merged**